docs/index.html

```diff
@@ -42,6 +42,7 @@ <h2 class="title">A 1x Engineer...</h2>
         <li>Appreciates code reviews from their peers that are constructive and helpful.</li>
         <li>Is open to learning.</li>
         <li>Asks for help when they need it.</li>
+        <li>Double-checking (or quadruple-checking?) their running code in their local machines.</li>
```

Comment:
For verb consistency! 😄 

```suggestion
        <li>Double-checks (or quadruple-checks?) their running code in their local machines.</li>
```

Reply:
Ah, thanks mate!